client/src/components/base/Input/index.tsx

```diff
@@ -27,6 +30,7 @@ const Input = ({
   placeholder,
```

Comment:
```suggestion
  placeholder = '',
```

이 부분에서 default value를 설정해줘도 좋을 것 같군요